src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -2138,6 +2138,25 @@ public void GetAsync_ServerNeedsAuthAndNoCredential_StatusCodeUnauthorized()
             });
         }
 
+        [OuterLoop]
+        [Fact]
+        public async Task GetAsync_IdnHostName()
+        {
+            HttpClientHandler handler = CreateHttpClientHandler();
+            using (var client = new HttpClient(handler))
+            {
+                /*
+                    international version of the Starbucks website
+                    Punnycode: xn--oy2b35ckwhba574atvuzkc.com
+                */
```

Comment:
In corefx, we usually do comment like this:

// Here is the comment.

Reply:
done